development/git-workflow/README.md

```diff
@@ -51,5 +51,56 @@ In all other branches we'll use a build number within a tagged commit to designa
 3. Create a commit with updated deliverables.
 4. Tag that commit with a build number.
 
+# Build Process for Creating a QA build
+
+Prerequisite: create qa branches on alpheios-core, embed-lib and webextension and push them upstream. 
+
+1. alpheios-core: merge master to qa
+2. alpheios-core: run `npm install && npm update`
+3. alpheios-core: run `npm tagged-commit`
+4. alpheios-core run `git push && git push --tag`
+5. embed-lib: merge master to qa
+6. embed-lib: run `npm install && npm update`
+7. embed-lib: run `npm tagged-commit`
+8. embed-lib run `git push && git push --tag`
+9. webextension: merge master to qa
+10. webextension: run `npm install && npm update`
+11. webextension: run `npm tagged-commit`
+12. webextension: run `git push && git push --tag`
+
+After these steps, we should have the following
+1. new packages of alpheios-components and alpheios-embedded on NPM with the `@qa` tag
+2. GitHub releases of alpheios-core, webextension and embed-lib in GitHub
+
+Running `tagged-commit` in __alpheios-core__ automates these steps:
+1. increases the build number and exports it to an environment variable which webpack injects into the code
```

Comment:
A minor correction here and in other similar places: the build number is passed to a `node-build` build script as a parameter, and then the build scripts injects it into a code. We're not using environment vars here.

Reply:
great thanks, will correct that.